agreements.md

```diff
@@ -176,6 +176,32 @@ This solution is meant to allow the team to auto-calibrate and recognize what co
 If the offensive comment is seen as severe and above the grade of this agreement, it will be raised to the HR circle to decide on further steps.
 Fortunately, so far, there have been no such situations.
 
+## Constructive feedback
+
+We aim to maintain a positive work environment where feedback is given constructively to promote growth and improvement.
+To manage situations where feedback might come across as negative or aggressive, we will use the following procedure:
+
+- If someone feels that feedback is too negative or aggressive towards any staff member, they should say:
+  “Could you please rephrase your concern in a more constructive way?”
+- The person who provided the feedback should then respond with:
+  “Sorry” and rephrase their feedback in a more constructive manner.
+- No further comments or defenses should be made by others present.
```

Comment:
```suggestion
```

this works for joke agreement, but doesn't work in what is supposed to be a discussion.
see example 1 - it ends with a question, so obviously more comments will follow